docs/attribution/0001-analytics.rst

```diff
@@ -8,273 +8,226 @@
-        type: 'banner',
-        action: 'clickthrough',
-        name: '<banner name>', //ex. Fb-Video-Compat
-    });
+Some form submissions are also being collected but newsletter signups are not. `(See Bug #13348)`_ They are instead
+being tracked with the ``newsletter_subscribe`` event.
 
-When a banner is dismissed:
+See the full list of events GA collects by default. https://support.google.com/analytics/answer/9216061
 
-.. code-block:: javascript
+Data Attributes
+~~~~~~~~~~~~~~~
 
-    // UA
-    dataLayer.push({
-        'eLabel': 'Banner Dismissal',
-        'data-banner-name': '<banner name>', //ex. Fb-Video-Compat
-        'data-banner-dismissal': '1',
-        'event': 'in-page-interaction'
-    });
-    // GA4
-    window.dataLayer.push({
-        event: 'widget_action',
-        type: 'banner',
-        action: 'dismiss',
-        name: '<banner name>' //ex. Fb-Video-Compat
-    });
+One place where we do still rely on GTM to watch for an trigger events is when we use data-attributes to pass
+information. There are only two events we do this with and they are primarily used on buttons and links.
 
 
-A/B tests
-~~~~~~~~~
+CTA Click
+"""""""""
 
-.. code-block:: javascript
+CTA ("Call to Action") click is intented to track the one or two main actions the page is designed to get the user
+to do. This data-attribute can be used on either ``<button>`` or ``<a>``. Links can go to or away from the site,
+a button might trigger a JS function instead of going away from the page, that's still a cta.
 
-    if (href.indexOf('v=a') !== -1) {
-        // UA
-        window.dataLayer.push({
-            'data-ex-variant': 'de-page',
-            'data-ex-name': 'Berlin-Campaign-Landing-Page'
-        });
-        // GA4
-        window.dataLayer.push({
-            event: 'experiment_view',
-            id: 'Berlin-Campaign-Landing-Page',
-            variant: 'de-page',
-        });
-    } else if (href.indexOf('v=b') !== -1) {
-        // UA
-        window.dataLayer.push({
-            'data-ex-variant': 'campaign-page',
-            'data-ex-name': 'Berlin-Campaign-Landing-Page'
-        });
-        // GA4
-        window.dataLayer.push({
-            event: 'experiment_view',
-            id: 'Berlin-Campaign-Landing-Page',
-            variant: 'campaign-page',
-        });
-    }
+The attribute ``data-cta-text`` must be present to trigger the event. All links to accounts.firefox.com must also use cta-type.
 
-GTM listeners & data attributes
--------------------------------
++-----------------------+----------------------------------------------------------------------------------+
+| Data Attribute        | Expected Value                                                                   |
++=======================+==================================================================================+
+| ``data-cta-text`` *   | Text or name of the link (e.g. ``Sign Up``, ``Start Here``, ``Get Relay``,       |
```

Comment:
**Question:** What is this asterisks referring to?

Reply:
It's a required attribute. I added a note in the docs.